aptos-move/framework/aptos-stdlib/sources/cryptography/curves.move

```diff
@@ -0,0 +1,217 @@
+module aptos_std::curves {
```

Comment:
A really bad news I have is that every native function you introduce in this module will have to be feature-gated: e.g., see `verify_range_proof` in the [Bulletproof PR](https://github.com/aptos-labs/aptos-core/pull/3444/files#diff-c7c797ab10d62523a98be9c78d5f6b3e54db76c38bc4fa89308d014169d9d091R66)